packages/origin-backend/migrations/1604562378046-SwaggerChanges.ts

```diff
@@ -0,0 +1,19 @@
+import { MigrationInterface, QueryRunner } from 'typeorm';
+
+export class SwaggerChanges1604562378046 implements MigrationInterface {
+    name = 'SwaggerChanges1604562378046';
+
+    public async up(queryRunner: QueryRunner): Promise<void> {
+        await queryRunner.query(`ALTER TABLE "organization_invitation" DROP COLUMN "status"`);
+        await queryRunner.query(
+            `ALTER TABLE "organization_invitation" ADD "status" character varying NOT NULL`
+        );
```

Comment:
This will fail on existing environments if a drop won't happen.

Reply:
Yeah, we need a default.

Reply:
I actually didn't change this. Someone else forgot to update migrations. 🤷